packages/ckeditor5-engine/src/view/domconverter.js

```diff
@@ -1546,7 +1547,9 @@ export default class DomConverter {
 	 * @returns {Boolean}
 	 */
 	_shouldRenameElement( elementName ) {
-		return this.renderingMode == 'editing' && elementName.toLowerCase() == 'script';
+		const name = elementName.toLocaleLowerCase();
```

Comment:
Why `toLocaleLowerCase()`? Is this backed by tests?

Reply:
Crap. Stupid autocorrect! Of course it should be `toLowerCase()` 🤦